queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+      SELECT * FROM animals WHERE name like 'mon'
```

Comment:
- [ ] Since the question specifically asks for names **ending** in "mon" I'd suggest updating your query to '%mon'.
- [ ] I also noticed your screenshot doesn't include results from all the queries so pelase make sure to take screenshots of all the 8 queries.